### PR TITLE
add: field for allowed sample rates

### PR DESCRIPTION
**Pull request by aQib-jahan**

### CODE REVIEW

**Review by shahruk10 — COMMENTED** [review]
A suggestion the docstring, but looks good otherwise

File: proto/cobaltspeech/transcribe/v5/transcribe.proto [hunk 1]
```diff
@@ -387,6 +387,9 @@ message ModelAttributes {
 
   // Attributes specifc to supporting recognition context.
   ContextInfo context_info = 2;
+
+  // Audio sample rates other than the model's native sample rate that can be accommodated
```
Comment (ppcssn) [inline proto/cobaltspeech/transcribe/v5/transcribe.proto:391]: It'd be nice to explicitly refer to `sample_rate` for native sample rate comment here and also in `sample_rate` comment (L385)

File: proto/cobaltspeech/transcribe/v5/transcribe.proto [hunk 2]
```diff
@@ -387,6 +387,9 @@ message ModelAttributes {
...
+  repeated uint32 allowed_sample_rates = 3;
```
Comment (ppcssn) [inline proto/cobaltspeech/transcribe/v5/transcribe.proto:392]: maybe `supported_sample_rates` ?

File: proto/cobaltspeech/transcribe/v5/transcribe.proto (hunk 1 above)
Comment (ppcssn) [inline proto/cobaltspeech/transcribe/v5/transcribe.proto:391]: we exclude native sample rate from this list?

Comment (ppcssn) [reply proto/cobaltspeech/transcribe/v5/transcribe.proto:391]: just asking, but do we handle them (native and those in this list) differently?

(if not, we can also include the native sample rate here, so it easier to loop all of them)

**Review by ppcssn — COMMENTED** [review]
I think this is okay to merge as it is. I tried thinking about other idea we can do and left them in the comments. I don't have strong opinion whether it good or bad on those idea, so feel free to skip them as you see fit.

File: proto/cobaltspeech/transcribe/v5/transcribe.proto (hunk 1 above)
Comment (aQib-jahan) [reply proto/cobaltspeech/transcribe/v5/transcribe.proto:391]: In the model-config file it doesn't include the native sample rate but in the error message the native one is included.

**Review by happyalu — APPROVED** [review]

**Pull request merged**